tools/geps/main.go

```diff
@@ -29,48 +29,49 @@ import (
 	"sort"
 	"strconv"
 	"strings"
+	"text/template"
 
 	"sigs.k8s.io/yaml"
 
 	gep "sigs.k8s.io/gateway-api/pkg/gep"
 )
 
+//go:embed templates/*.tmpl
+var templateFile embed.FS
+
 var (
-	GEPSDir        string
-	MKDocsTemplate string
-	SkipGEPNumber  string
+	GEPSDir       string
+	SkipGEPNumber string
 )
 
 // Those are the GEPs that will be included in the final navigation bar
 // The order established below will be the order that the statuses will be shown
 var includeGEPStatus = []gep.GEPStatus{
+	gep.GEPStatusProvisional,
+	gep.GEPStatusPrototyping,
 	gep.GEPStatusImplementable,
 	gep.GEPStatusExperimental,
 	gep.GEPStatusStandard,
 	gep.GEPStatusMemorandum,
 }
```

Comment:
On looking at the page, I think this should go in descending order of stability, so should be:

* Standard
* Memorandum
* Experimental
* Implementable
* Prototyping
* Provisional

Similarly for the List page, that should go Standard, Memorandum, Experimental, Implementable, Prototyping, Provisional, and the default start page should be Standard.

Basically, I think that we should be trying to show, by default, how many things are stable already.